sdk/cosmosdb/cosmos/src/bulk/BulkStreamerCache.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Do we really need this cache? We can always create a new streamer everytime Cx calls `items.getBulkStreamer()`